wfe/wfe.go

```diff
@@ -1444,7 +1444,12 @@ func (wfe *WebFrontEndImpl) deactivateAuthorization(ctx context.Context, authz *
 		wfe.sendError(response, logEvent, probs.Malformed("Invalid status value"), err)
 		return false
 	}
-	err = wfe.RA.DeactivateAuthorization(ctx, *authz)
+	pbAuthz, err := grpc.AuthzToPB(*authz)
```

Comment:
This whole `deactivateAuthorization` helper function is only called from one place (`Authorization`), which already has access to both the `authz` and the `authzPB`. Just change this function's signature to take the `authzPB` directly so we don't have to convert back and forth multiple times.